RocketChat/Rocket.Chat.ReactNative
# [FIX] Permissions to edit livechat when the user is a livechat-agent

## Proposed changes

Last month the way that the API permit a user to edit the `livechat` was changed to `'edit-omnichannel-contact'`, because that was occurring a bug when the agent wants to edit something.

## Issue(s)	

The same agent could edit a `livechat` on the web, but on the mobile app couldn't

## How to test or reproduce

1 - Activate Omnicahnnel ->  https://docs.rocket.chat/guides/omnichannel-guides/omnichannel-admins-guide
3 - Start a new chat through Livechat https://multiverse.rocket.chat/livechat
4 - Go to Omnichannel Queue (Mobile)
5 - Open an incoming chat
6 - Click on the title
7- At `Actions screen` click on name of the user
8 - Will appear the `edit button`

## Screenshots
<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</thead>
</tr>
<body>
<tr>
<td>
<img src="https://user-images.githubusercontent.com/47038980/126845957-09cd9128-2eca-442f-ad62-796ca01cf29b.png"  height="450"/>
</td>
<td>
<img src="https://user-images.githubusercontent.com/47038980/126845970-53c70ed3-ef56-49c9-812f-3061c57b49ac.png" height="450" />
</td>
</tr>
</tbody>
</table>

## Types of changes




## Checklist



## Further comments
